AnkiDroid/src/main/java/com/ichi2/utils/ImportUtils.java

```diff
@@ -141,11 +141,11 @@ private static void sendShowImportFileDialogMsg(String path) {
     }
 
     public static boolean isCollectionPackage(String filename) {
-        return filename != null && (filename.toLowerCase().endsWith(".colpkg") || filename.equals("collection.apkg"));
+        return filename != null && (filename.toLowerCase().endsWith(".colpkg") || filename.toLowerCase().endsWith(".colpkg.zip") || filename.equals("collection.apkg"));
```

Comment:
Actually, for completeness shouldn't the full name check also handle it?

```suggestion
        return filename != null && (filename.toLowerCase().endsWith(".colpkg") || filename.toLowerCase().endsWith(".colpkg.zip") || (filename.equals("collection.apkg") || filename.equals("collection.apkg.zip"));
```

Similarly below. And then codacy will likely grumble about the static string not being first...